src/main/steps/applicant1/check-your-answers/content.ts

```diff
@@ -654,8 +658,8 @@ const cy: typeof en = ({
       line1: `${userCase.inTheUk.replace('Yes', 'Do').replace('No', 'Naddo')}`,
```

Comment:
Can we use stripTags here too?

Reply:
I didn't think it was necessary for YesOrNo fields

Reply:
Ah yeah, I was just comparing to L225 but I agree it's not needed.